nodeup/pkg/model/kubelet.go

```diff
@@ -240,6 +258,8 @@ func (b *KubeletBuilder) buildSystemdService() *nodetasks.Service {
 	manifest.Set("Service", "StartLimitInterval", "0")
 	manifest.Set("Service", "KillMode", "process")
 	manifest.Set("Service", "User", "root")
+	manifest.Set("Service", "CPUAccounting", "true")
+	manifest.Set("Service", "MemoryAccounting", "true")
```

Comment:
Is this related to the manifest directory change?  If not, let's split it out so that we can review the ideas separately.  This isn't as clear to me as the directory change, so it will delay the directory change.

Reply:
Not a problem.  I just reverted that commit and re-synced my fork's feature branch with kops' master branch.